### PR TITLE
fix: Guide component mobile overflow

### PR DESCRIPTION
**Pull request by 0x7h11s**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #309 

## What is the new behavior?

Guide component mobile end modification of default values

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/theme-saas/src/guide/index.less [hunk 1]
```diff
@@ -249,12 +249,10 @@
     opacity: 0;
     visibility: hidden;
     z-index: 9999;
-    width: 510px;
+    width: var(--tiny-guide-shepherd-element-size);
```
Comment (zzcr) [inline packages/theme-saas/src/guide/index.less]: theme-saas里面的样式不支持使用--tiny-guide-shepherd-element-size，所以theme-saas文件可以不用改

Comment (kagol) [reply packages/theme-saas/src/guide/index.less]: @xlearns 可以移除 theme-sass 里面的主题变量

**Review by kagol — COMMENTED** [review]
@xlearns 有空记得处理下检视意见哈

File: packages/theme-saas/src/guide/index.less (hunk 1 above)
Comment (0x7h11s) [reply packages/theme-saas/src/guide/index.less]: ok

**Comment by 0x7h11s** [comment]
大佬，非常抱歉，由于之前没弄过，自己的操作不当，给我的pr记录弄丢了，我又重新提交了一个pr。&nbsp;


?&nbsp;?
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年10月9日(星期一) 晚上10:13
收件人: ***@***.***&gt;; 
抄送: "‭ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [opentiny/tiny-vue] fix: Guide component mobile overflow (PR #499)





 
@kagol commented on this pull request.
 
@xlearns 有空记得处理下检视意见哈

**Pull request closed without merging**